### PR TITLE
New Domains Added By Nandu (i am back)

### PR DESCRIPTION
**Pull request by NanduWasTaken**
All domains had gone through Google Transparency Check and shows that they are unsafe.
Except 2 domains which still had sketchy redirects to unsafe websites.

### CODE REVIEW

**Comment by NanduWasTaken** [comment]
By The way all the domain extention are starting by letter A 
I will probably will make a another pull request by the domain extention starting with letter B tommorow

**Comment by nikolaischunk** [comment]
Can you provide a proof of those domains + the source where you got it from?

**Comment by nikolaischunk** [comment]
Also those are just discord.sometld domains? I mean sure list every possible TLD and append it to discord. ?

File: domain-list.json [hunk 1]
```diff
@@ -926,6 +926,7 @@
     "cupcsgo.ru",
     "cupgoo.xyz",
     "cupwin.xyz",
+    "cyberspoort.site",
```
Comment (nikolaischunk) [inline domain-list.json]: Already added

**Review by nikolaischunk — APPROVED** [review]

File: domain-list.json (hunk 1 above)
Comment (NanduWasTaken) [reply domain-list.json]: Yeah the cyberspoort.site domain got deleted in my pull request so I thought I somehow put it there. In the end i released the bot was doing it 😅

**Comment by NanduWasTaken** [comment]
> Can you provide a proof of those domains + the source where you got it from?

I take up the list of the TDLs from ![here] (https://www.domain.com/domains/new-domain-extensions)
And manually go to the domains if my browser shows like this
Screenshot_2022-10-05-13-39-08-423
Then I add it to the list (if it shows like that it means that it has failed the Google Transparency Check)

**Pull request merged**